assets/js/types/type-defs/payments.ts

```diff
@@ -114,6 +114,7 @@ export interface ExpressPaymentMethodConfigInstance {
 	content: ReactNode;
 	edit: ReactNode;
 	paymentMethodId?: string;
+	placeOrderButtonLabel: string;
```

Comment:
This should be optional here, as not all payment methods will define a custom label for the button.
```suggestion
	placeOrderButtonLabel?: string;
```

Reply:
Thanks for suggesting this, @alexflorisca.